lib/passport-wsfed-saml2/saml.js

```diff
@@ -21,19 +22,17 @@ var SAML = function (options) {
   }
 
   this.options.checkExpiration = (typeof this.options.checkExpiration !== 'undefined') ? this.options.checkExpiration : true;
+  // We need to enable checkCertExpiration gradually so as not to break everyone who doesn't know (or care) their cert is expired
+  this.options.checkCertExpiration = (typeof this.options.checkCertExpiration !== 'undefined') ? this.options.checkCertExpiration : false;
   this.options.checkAudience = (typeof this.options.checkAudience !== 'undefined') ? this.options.checkAudience : true;
   this.options.checkRecipient = (typeof this.options.checkRecipient !== 'undefined') ? this.options.checkRecipient : true;
   this.options.checkNameQualifier = (typeof this.options.checkNameQualifier !== 'undefined') ? this.options.checkNameQualifier : true;
   this.options.checkSPNameQualifier = (typeof this.options.checkSPNameQualifier !== 'undefined') ? this.options.checkSPNameQualifier : true;
   this.eventEmitter = this.options.eventEmitter || new EventEmitter();
 };
 
-SAML.prototype.certToPEM = function (cert) {
-  cert = cert.match(/.{1,64}/g).join('\n');
-  cert = "-----BEGIN CERTIFICATE-----\n" + cert;
-  cert = cert + "\n-----END CERTIFICATE-----\n";
-  return cert;
-};
+// TODO - update all calls to this to simply use utils.
+SAML.prototype.certToPEM = utils.certToPEM;
```

Comment:
Rather than the TODO, let's just go ahead and replace these assuming the tests are in place.

Reply:
Done